aptos-move/aptos-vm/src/zkid_validation.rs

```diff
@@ -0,0 +1,146 @@
+// Copyright © Aptos Foundation
+// Parts of the project are originally copyright © Meta Platforms, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::move_vm_ext::{AptosMoveResolver, SessionExt};
+use aptos_types::{
+    jwks::{jwk::JWK, PatchedJWKs},
+    on_chain_config::{CurrentTimeMicroseconds, OnChainConfig},
+    transaction::SignedTransaction,
+    vm_status::{StatusCode, VMStatus},
+    zkid::{ZkpOrOpenIdSig, MAX_ZK_ID_AUTHENTICATORS_ALLOWED},
+};
+use aptos_vm_logging::log_schema::AdapterLogSchema;
+use move_binary_format::errors::Location;
+use move_core_types::{language_storage::CORE_CODE_ADDRESS, move_resource::MoveStructType};
+
+fn get_current_time_onchain(
+    resolver: &impl AptosMoveResolver,
+) -> anyhow::Result<CurrentTimeMicroseconds, VMStatus> {
+    CurrentTimeMicroseconds::fetch_config(resolver).ok_or_else(|| {
+        VMStatus::error(
+            StatusCode::VALUE_DESERIALIZATION_ERROR,
+            Some("could not fetch CurrentTimeMicroseconds on-chain config".to_string()),
+        )
+    })
+}
+
+fn get_jwks_onchain(resolver: &impl AptosMoveResolver) -> anyhow::Result<PatchedJWKs, VMStatus> {
```

Comment:
I thought we were gonna call the `try_get_jwk_by_issuer` function in `jwks.move`